cloudinary-core/src/main/java/com/cloudinary/Api.java

```diff
@@ -240,6 +240,15 @@ public ApiResponse deleteResources(Iterable<String> publicIds, Map options) thro
         return callApi(HttpMethod.DELETE, Arrays.asList("resources", resourceType, type), params, options);
     }
 
+    public ApiResponse deleteResourcesByAssetIds(Iterable<String> assetIds, Map options) throws Exception {
+        if (options == null) options = ObjectUtils.emptyMap();
+        String resourceType = ObjectUtils.asString(options.get("resource_type"), "image");
+        String type = ObjectUtils.asString(options.get("type"), "upload");
```

Comment:
resourceType and type are redundant.
See:
https://github.com/cloudinary/pycloudinary/commit/fe916055eb747c4b6c3c122eebc58e84a1d884c5#diff-d70a14183661deecc48fa77958d4f2a1638993fceb549658504b0c0948627895R313